app/vmselect/promql/rollup.go

```diff
@@ -681,6 +681,8 @@ func (rc *rollupConfig) DoTimeseriesMap(tsm *timeseriesMap, values []float64, ti
 	return samplesScanned
 }
 
+const defaultScrapeInterval = 5 * 60 * 1000
```

Comment:
I believe the `app/vmselect/promql` package should already contain a constant equal to 5 minutes. If such constant exists there, can we re-use it instead of defining new one?

Reply:
Reverted.